docs/odoo/usage/install-module.rst

```diff
@@ -23,7 +47,7 @@ install
 * :doc:`activate developer mode <debug-mode>`
 * navigate to ``Apps`` menu
 * click ``Update Apps List``
-* click ``Apps`` in left side bar
+* click ``Apps`` 
```

Comment:
After clicking  ``Update Apps List`` there is no ``Apps`` button. It may confuse